src/errorHandler.ts

```diff
@@ -15,9 +15,21 @@ let statusBarItem: StatusBarItem;
 let outputChannel: OutputChannel;
 let prettierInformation: string;
 
+function showPrettierErrorMessage() {
+    const showErrorAction = 'Show error';
+
+    window
+        .showErrorMessage('Prettier failed to format!', showErrorAction)
```

Comment:
What about putting an other message in there ? Parameter? I see us use it a little more in the future if this new notification center fits out needs.

Quick idea: `addToOuput` shows a notification with a short message. `More` button to open the panel with the complete error message ? What do you think?